symengine/basic-inl.h

```diff
@@ -103,6 +103,15 @@ hash_t vec_hash<T>::operator()(const T &v) const
     return h;
 }
 
+//! workaround for MinGW bug
+template <typename T>
+std::string to_string(const T &value)
+{
+    std::ostringstream ss;
+    ss << value;
+    return ss.str();
+}
+
```

Comment:
Is the bug that `std::to_string` is not available in MinGW? For now this is good enough, later perhaps we can somehow inject it into the `std` namespace, just for MinGW, perhaps cmake can figure this out.

Reply:
Yes. `std::to_string` is not available on MinGW.